src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/DescendantTokenCheck.xml

```diff
@@ -16,6 +16,9 @@
  about language structures.
  &lt;/p&gt;</description>
          <properties>
+            <property default-value="" name="tokens" type="allTokenTypes">
+               <description>All checkstyle token types.</description>
```

Comment:
this should be a description of property, not a description of type.

please use `tokens to check`